projects/ion/src/lib/alert/alert.component.ts

```diff
@@ -0,0 +1,32 @@
+import { Component, Input } from '@angular/core';
+
+export type AlertType = 'success' | 'info' | 'alert' | 'danger';
```

Comment:
Please, create this type in core.

**status.ts**
```suggestion
export type StatusType = 'success' | 'info' | 'alert' | 'danger';
```

and call in this document